system_metrics_collector/src/system_metrics_collector/proc_cpu_data.cpp

```diff
@@ -58,4 +58,16 @@ bool ProcCpuData::isMeasurementEmpty() const
   return cpu_label == ProcCpuData::EMPTY_LABEL;
 }
 
+/*static*/ constexpr const uint64_t ProcPidCpuData::EMPTY_DATA;
```

Comment:
Prefer constant definitions together (at the top)

Reply:
Done.